DatadogLogs/Sources/Logs.swift

```diff
@@ -0,0 +1,110 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import Foundation
+import DatadogInternal
+
+/// iOS Log Collection
+///
+/// Send logs to Datadog from your iOS applications with Datadog’s dd-sdk-ios client-side logging library and leverage the following features:
+/// - Log to Datadog in JSON format natively.
+/// - Use default and add custom attributes to each log sent.
+/// - Record real client IP addresses and User-Agents.
+/// - Leverage optimized network usage with automatic bulk posts.
+public struct Logs {
+    /// The Logs general configuration.
+    ///
+    /// This configuration will be applied to all Logger instances.
+    public struct Configuration {
+        public typealias EventMapperClosure = (LogEvent) -> LogEvent?
+        /// Sets the sampling rate for logging.
+        ///
+        /// The sampling rate must be a value between `0` and `100`. A value of `0` means no logs will be processed, `100`
+        /// means all logs will be processed.
+        ///
+        /// By default sampling is disabled, meaning that all logs are being processed).
+        public var sampleRate: Float
+
+        /// Overrides the custom server endpoint where Logs are sent.
+        public var customEndpoint: URL?
+
+        /// Overrides the main bundle instance.
+        public var bundle: Bundle = .main
```

Comment:
suggestion/ Seeing this in Logs public API seems odd, so I checked if we can remove it. Looks we can:
```diff
// LogEventBuilder.swift

- loggerName: loggerName,
+ loggerName: loggerName ?? context.applicationBundleIdentifier,
```
Otherwise, the user will have to configure `bundle: Bundle` both in Core and Logs, which can be quite error-prone.

WDYT @maxep ? If you agree, we can move this update to separate JIRA for later (before GA).

Reply:
Oh nice, lets do it here, it's a small impact 👍